libcst/codemod/visitors/_apply_type_annotations.py

```diff
@@ -0,0 +1,558 @@
+# Copyright (c) 2016-present, Facebook, Inc.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree
+
+
```

Comment:
does this still work if pyre-strict?

Reply:
No, I have to add it. Thanks for catching that.

Side note: I notice that the `.pyre_configuration` file was [recently](https://github.com/Instagram/LibCST/commit/ea8619ade5eecaf02136be1d588ee075145097c8) moved into the .circleci directory. I don't know if that was intentional, but I had to copy-paste the configuration file into the root directory for local testing.

Also, given that almost all the files are marked `pyre-strict`, you could make them strict by default using `"strict": true`.

Reply:
Yes, it's intentional, we updated our README and ask developer to follow the steps to copy `.pyre_configuration.example` as `.pyre_configuration` as well as adding custom python package path.
Each developer may have a venv in different location and we want to include the venv site-packages path in the config. So Pyre parses all installed Python packages and can understand more types from more packages.
https://github.com/Instagram/LibCST/blob/master/README.rst